cmd/internal/flag.go

```diff
@@ -107,9 +107,9 @@ func initRegistryClientFlags() {
 }
 
 func initImageVerifyCacheFlags() {
-	flag.BoolVar(&imageVerifyCacheEnabled, "imageVerifyCacheEnabled", true, "Whether to use a TTL cache for storing verified images.")
-	flag.Int64Var(&imageVerifyCacheMaxSize, "imageVerifyCacheMaxSize", 0, "Max size limit for the TTL cache, 0 means no size limit.")
-	flag.Int64Var(&imageVerifyCacheTTLDuration, "imageVerifyCacheTTLDuration", 0, "Max TTL value for a cache, 0 means no TTL.")
+	flag.BoolVar(&imageVerifyCacheEnabled, "imageVerifyCacheEnabled", false, "Whether to use a TTL cache for storing verified images.")
+	flag.Int64Var(&imageVerifyCacheMaxSize, "imageVerifyCacheMaxSize", 0, "Max size limit for the TTL cache, 0 means default 1000 size limit.")
+	flag.Int64Var(&imageVerifyCacheTTLDuration, "imageVerifyCacheTTLDuration", 0, "Max TTL value for a cache, 0 means default 1 hour TTL.")
```

Comment:
Why put the default as 0? Can't we set defaults directly?

Reply:
I was handling default cases during cache creation. @vishal-chdhry should we set default values here??

Reply:
yes. as it say default are 1000 and 1 hrs, they should be set as such too

Reply:
done